<launch>
	<!-- serial communication with the Tipper -->
	 <node pkg="serial_nmea" name="serial_nmea_sensor1" type="serial_nmea_node" respawn="true" output="screen" >
		<param name="nmea_from_device_pub" value="/fmData/nmea_from_tipper"/>
		<param name="nmea_to_device_sub" value="/fmData/nmea_to_tipper"/>
                <param name="serial_device" value="/dev/tipper"/>
		<param name="serial_baudrate" value="9600"/>
		<param name="use_nmea_checksum" value="true"/>
         </node>
 

	<node pkg="tipper_unit" type="sdu_rsd_tipper_node" name="tipper" output="screen">
		<param name="tipper_nmea_sub" value="/fmData/nmea_from_tipper" />
		<param name="tipper_nmea_pub" value="/fmData/nmea_to_tipper" />
		<param name="tipper_done" value="/tip_done" />
		<param name="tipper_start" value="/tip_start" />
	</node>
</launch>
